delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/indexing/ElasticSearchSink.scala

```diff
@@ -38,16 +38,30 @@ final class ElasticSearchSink(
 
   override def inType: Typeable[Json] = Typeable[Json]
 
+  /**
+    * Creates an id to use for a bulk operation
+    * @param elem
+    *   the elem for which to create the ID
+    * @return
+    *   an id that incorporates the project (if available), the elem id, and the revision
+    */
+  private def bulkOpId(elem: Elem[_]): String = {
+    elem.project match {
```

Comment:
It also depends if we index a state or an event
For a state, we don't want to add the revision as we want to overwrite the previous one